lib/database/models/qcFlagType.js

```diff
@@ -44,6 +44,11 @@ module.exports = (sequelize) => {
                     is: /^#[0-9a-fA-F]{6}$/i,
                 },
             },
+            mcReproducible: {
+                type: Sequelize.BOOLEAN,
+                allowNull: true,
```

Comment:
Do we want to allowNull or instead use a `default value` of false?
At least in the seeders for dev and tst, I see you always placed `false`

Reply:
In current API, it's not important, it's important whether it's true or not-true (false or null), for seeders I've just chosen one of two possible values